src/platforms/apple/troubleshooting.mdx

```diff
@@ -19,3 +19,7 @@ We recommend updating to at least [7.11.0](https://github.com/getsentry/sentry-c
 ## Crashes with HTTP Auto Performance Instrumentation
 
 We recommend updating to at least [7.5.3](https://github.com/getsentry/sentry-cocoa/releases/tag/7.5.3), because the HTTP instrumentation can lead to crashes. Alternatively, you can also <PlatformLink to="/performance/instrumentation/automatic-instrumentation/#http-instrumentation">disable the feature</PlatformLink>.
+
+## Duplicated crashes event with screenshots
+
+If you opt-in for screenshots for crash, we recommend to update to at least [7.25.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.5.3). Since the introduction of screenshots for crash in the version 7.20.0, projects that enabled 'enableCaptureScreenshot' had their crash reports duplicated.
```

Comment:
I would change the wording a bit and point to the correct releases.
```suggestion
Since the introduction of screenshots for crashes in version [7.20.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.20.0), projects with  'enableCaptureScreenshot' enabled received duplicated crash reports. Therefore, we recommend updating to at least [7.25.0](https://github.com/getsentry/sentry-cocoa/releases/tag/7.25.0) to fix this problem.
```